js/window.js

```diff
@@ -7,7 +7,7 @@ window.open = (...args) => {
   let w = originalOpen.apply(this, args);
    // Try catch for cross domain safeguard
   if(w && !w.name.includes('Notifications') && w.name !== 'queueCounter' && args[1] !== 'main') {
-    let stream = args[0].split('&')[1];
+    let stream = decodeURIComponent(args[0].split('&')[1]);
```

Comment:
while we are in here, lets try to do a better job of extracting the GET / URL args... if the arg order changed we will have to modify this again

(Code smell: magic number 1)

Reply:
👍 (but lets revisit post merge 😄 )